app/fantasy_cricket/__init__.py

```diff
@@ -0,0 +1,20 @@
+"""
+Module to set path of files in the directory
+Copyright (C) 2020  Royston E Tauro & Sammith S Bharadwaj & Shreyas Raviprasad
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+"""
```

Comment:
Might find it easier to have a single license for the entire repo

Reply:
Actually AGPL 3.0 had suggested to do it for each file that's why I had done it
I don't want to take it out as it was suggested by the LICENSE itself